apps/code/python_toolbox.cpp

```diff
@@ -145,6 +145,7 @@ const ToolboxMessageTree TurtleModuleChildren[] = {
   ToolboxMessageTree::Leaf(I18n::Message::PythonTurtleCommandShowturtle, I18n::Message::PythonTurtleShowturtle, false),
   ToolboxMessageTree::Leaf(I18n::Message::PythonTurtleCommandHideturtle, I18n::Message::PythonTurtleHideturtle, false),
   ToolboxMessageTree::Leaf(I18n::Message::PythonTurtleCommandColor, I18n::Message::PythonTurtleColor, false, I18n::Message::PythonTurtleCommandColorWithoutArg),
+  ToolboxMessageTree::Leaf(I18n::Message::PythonTurtleCommandColorMode, I18n::Message::PythonTurtleColorMode, false, I18n::Message::PythonTurtleCommandColorModeWithoutArg),
```

Comment:
Remove PythonTurtleCommandColorModeWithoutArg

Reply:
And remove the `false` to use the algorithm to remove the argument  "x" from `I18n::Message::PythonTurtleColorMode.`

It should thus be: `ToolboxMessageTree::Leaf(I18n::Message::PythonTurtleCommandColorMode, I18n::Message::PythonTurtleColorMode)`